app/helpers/priceFormatter.test.js

```diff
@@ -0,0 +1,14 @@
+import { formatPrice } from './priceFormatter';
+
+describe('formatPrice', () => {
+  it.each`
+    priceValue   | expected
+    ${10}        | ${'10.00'}
```

Comment:
shouldn't the price be 3 decimal places?

Reply:
This function is to format the price for display purposes. which is to always round down to the nearest 2 decimal place number.  We may get the price come back to us from the API in 3 decimal places